_layouts/menu.html

```diff
@@ -16,15 +16,15 @@
       <li class="submenu-item">
         <a
           class="menu-link"
-          href="https://github.com/tc39/agendas/blob/HEAD/2022/09.md"
+          href="https://github.com/tc39/agendas/blob/main/2023/03.md"
```

Comment:
```suggestion
          href="https://github.com/tc39/agendas/blob/HEAD/2023/03.md"
```

Reply:
I prefer the static ref over the symbolic

Reply:
The intention is to use links that don't hardcode the name of the default branch, to avoid the same churn that happened with master → main.

Reply:
`HEAD` is not actually documented anywhere and could break, although unlikely.

one subtle difference is that HEAD actually points to a commit id (the HEAD of course), and if you navigate around, all the links become commit-tree specific which is really annoying.  like if from there, I click the month, I go to `https://github.com/tc39/agendas/tree/9221e949320f570bc86405ec3a90310b219fb0bd/2023` instead of `https://github.com/tc39/agendas/tree/main/2023`

Reply:
I'm skeptical it's not documented anywhere, but either way all of TC39 relies on it already so I don't think that's a significant concern :-)

Re the commit ID, that's true, but why is that annoying?

Reply:
well -- for one example where it was annoying was updating these links.  I went to the existing links, and clicked to navigate the repo to go to the most recent, and of course I did not get the links I needed.  I got:

`https://github.com/tc39/agendas/blob/9221e949320f570bc86405ec3a90310b219fb0bd/2023/03.md`
and
`https://github.com/tc39/notes/tree/c7fa95d3f2d0af68eaba9d388d5f79f2e166a4cc/meetings/2023-01`

Reply:
Fair. I'm not sure that's worth hardcoding breakable links tho for the common use case of "reading the notes"

Reply:
they're not really breakable any more than the HEAD links are... even `master` links still redirect to HEAD.  (there are some in this repo that should get changed as well)

Reply:
Those redirects will break if anyone ever accidentally creates a master branch, altho the fix is easy (delete that branch) so maybe now that github has implemented this redirection logic (which didn't exist for awhile after people were switching default branch names en masse) it's not that big a deal 🤔

Reply:
![Screenshot 2023-03-06 at 1 35 58 PM](https://user-images.githubusercontent.com/15326526/223216849-eb050a42-5c13-43bb-937b-f1bd0c7339b0.png)

currently nearly a 50/50 split betwixt these variations in code refs anyway

not that it means anything

🤷

Reply:
i think that's mostly in proposal repos; in the long-term archival ones, like proposals and agendas and the spec, we're pretty good about sticking to HEAD i hope

Reply:
I'm not gonna die on this hill, so I'll change it but

Reply:
some of those refs should be pointing directly to commit IDs, I imagine, but... trade offs...

in any case, I will change to HEAD per momentum in this usage here, but for the record, suggest we use `main` in general

Reply:
I'll think more on it, thanks